test/ao/sparsity/test_pruner.py

```diff
@@ -288,18 +360,25 @@ def test_step_linear(self):
             for model in complex_models:
                 self._test_step_linear_on_device(model, False, torch.device(device))
 
-    def _test_step_conv2d_on_device(self, model, device):
+    def _test_step_conv2d_on_device(self, model, config, device):
         model = model.to(device)
         x = torch.ones((1, 1, 28, 28))
         pruner = SimplePruner(None)
-        pruner.prepare(model, None)
+        pruner.prepare(model, config)
         self._check_pruner_valid_before_step(model, pruner, device)
         pruner.step()
+        if type(model) is Conv2dBN:
+            assert model.seq[1].parametrizations.weight[0].pruned_outputs == model.seq[0].parametrizations.weight[0].pruned_outputs
+            assert model.bn.parametrizations.weight[0].pruned_outputs == model.conv2d.parametrizations.weight[0].pruned_outputs
         self._check_pruner_valid_after_step(model, pruner, {1}, device)
         assert model(x).shape == (1, 64, 24, 24)
 
     def test_step_conv2d(self):
-        models = [Conv2dA(), Conv2dB(), Conv2dC()]
+        bn_model = Conv2dBN()
+        bn_config = [(bn_model.seq[0], bn_model.seq[1]), (bn_model.conv2d, bn_model.bn)]
```

Comment:
what does this config mean?  Does it mean that `bn_model.seq[0]` and `bn_model.seq[1]` need to be pruned with the knowledge that they execute sequentially?